cloud/amazon/ec2_vpc_igw.py

```diff
@@ -116,8 +114,8 @@ def main():
     argument_spec = ec2_argument_spec()
     argument_spec.update(
         dict(
-            vpc_id = dict(required=True),
-            state = dict(choices=['present', 'absent'], default='present')
+            vpc_id = dict(required=True, default=None),
+            state = dict(required=False, default='present', choices=['present', 'absent'])
```

Comment:
Not sure of the reason behind this change either - `required=False` is default and everything else is just reordering